metasrv/src/meta_service/raftmeta.rs

```diff
@@ -503,17 +504,17 @@ impl MetaNode {
                 body: ForwardRequestBody::Join(JoinRequest {
                     node_id: conf.id,
                     endpoint: raft_api_advertise_host_endpoint.clone(),
+                    grpc_api_addr: grpc_api_addr.clone(),
```

Comment:
grpc_api_addr seems no more used, so can remove .clone()?

Reply:
This is an assignment in a for loop, so need to clone here.